scripts/slack/notify-failure-on-branch.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+NOTIFY_BRANCH_NAME="staging"
+DISPLAY_BRANCH_NAME="$1"
+
+if [[ "$CIRCLE_BRANCH" == "$NOTIFY_BRANCH_NAME" ]]; then
```

Comment:
Wondering if we could directly compare instead of using a variable
```suggestion
if [[ "$CIRCLE_BRANCH" == "staging" ]]; then
```

Reply:
we totally can, I've put it as a variable so if we want to parametrise it, it will be super easy, but I guess it will only always be staging?

Reply:
I think the script is really tiny and adding an extra variable makes it harder to read because now you have to go back and figure out what value it is assigned. A variable would make more sense if the pattern started repeating 🙏Just a suggestion, up to you if you want to change it!

Reply:
that makes sense, let's add it when we need it! 👍